zmon-controller-app/src/main/java/org/zalando/zmon/service/impl/ZMonServiceImpl.java

```diff
@@ -611,7 +613,7 @@ public Date getMaxCheckDefinitionLastModified() {
     }
 
     private ExecutionStatus buildStatus(int alertsActive, final Map<String, Response<Long>> queueSize,
-                                        final Map<String, Response<String>> lastUpdates, final Map<String, Response<String>> invocations) {
+                                        final Map<String, Response<String>> lastUpdates, final Map<String, Response<String>> invocations, final ServiceLevelStatus.ServiceLevelStatusData serviceLevelStatus) {
```

Comment:
`serviceLevelStatus` is not used (IDE should actually highlight that kind of mistakes, doesn't it?)